aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -590,7 +605,11 @@ def build_ome(
             )
             images.append(img)
 
-        ome_object = OME(creator=f"aicsimageio {get_module_version()}", images=images)
+        # NOTE:
+        # This version is updates as a part of bumpversion
+        # Do not change this manually
+        # We have to do this to avoid circular imports
+        ome_object = OME(creator="aicsimageio 4.0.0.dev3", images=images)
```

Comment:
this is way uglier than get_module_version...

Reply:
I know but to get `OmeTiffWriter` into `AICSImage` object gotta avoid the circular deps somehow... I feel like @AetherUnbound probably has a better idea. I trust her Python knowledge.

Reply:
What about a dedicated version file? That could be imported in the package `__init__.py`, but for this case it could potentially be imported directly. I agree that `get_module_version()` is probably the better choice here.

Reply:
I will play around with this a bit. `VERSION.txt` next to the init may be the next best option but I may also just move the `OmeTiffWriter` import _into_ the save function. I think that may dodge the issue.

Reply:
I like the latter alternative! Reduce import delay unless strictly necessary for the save

Reply:
Easily resolved by delaying the import of OmeTiffWriter to inside the save.